src/energyplus/ForwardTranslator/ForwardTranslatePlantEquipmentOperationSchemes.cpp

```diff
@@ -483,6 +491,9 @@ namespace energyplus {
         // As a result, this is handled in coolingComponents() and heatingComponents() directly
         return ComponentType::NONE;
       }
+      case openstudio::IddObjectType::OS_Chiller_Electric_ASHRAE205: {
+        return ComponentType::COOLING;
+      }
```

Comment:
No shenanigans needed, it'll always be on the supply side of only one Plant Loop.